cfme/tests/infrastructure/test_webmks_console.py

```diff
@@ -1,23 +1,17 @@
 """Test for WebMKS Remote Consoles of VMware Providers."""
-import imghdr
-import socket
-
 import pytest
-from wait_for import wait_for
 
-from cfme.infrastructure.provider import InfraProvider
+from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.markers.env_markers.provider import providers
-from cfme.utils import ssh
-from cfme.utils.conf import credentials
 from cfme.utils.generators import random_vm_name
-from cfme.utils.log import logger
 from cfme.utils.providers import ProviderFilter
 
 
 pytestmark = [
     pytest.mark.usefixtures('setup_provider'),
     pytest.mark.provider(gen_func=providers,
-                         filters=[ProviderFilter(classes=[InfraProvider],
+                         filters=[ProviderFilter(classes=[VMwareProvider],
```

Comment:
Don't actually need to use ProviderFilter here either, you can pass required_flags to the provider mark directly, and providers is the default gen method.

Reply:
Ok. I can update that.